### DISCUSSION TITLE
Is ScriptsLocation case-sensitive?

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1490] [Q&A] [answered]
Whatever's the answer, from which version please?

**Comment by kagikn** [comment, marked as answer]
it's case-insensitive in **literally all SHVDN versions**, because Windows file system treats file and directory names as case-insensitive by default (though [the file system can treat them as case-sensitive since Windows 10, build 17107](https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity)). Maybe not the case if the user have made files and directory names case-sensitive in the game directory, but I believe no one bothers to do this.

**Reply by Michael21107** [reply]
thanks a lot
